lprocwrapper.q

```diff
@@ -0,0 +1,18 @@
+/function which lets us call launchprocess.sh from inside a TorQ process
```

Comment:
should this script still be here? looks like it's maybe replaced by `launchprocess.q`?

Reply:
Yeah I changed the names to launchprocess.sh and launchprocess.q (instead of lprocwrapper.q). But actually since then I've added code for the wrapper for killproccess.sh into that same file, so as it stands there are launchprocess.sh, killprocess.sh and wrappers.q

Reply:
OK - just make sure that these changes get reflected on git :+1: